app/src/main/AndroidManifest.xml

```diff
@@ -9,13 +9,13 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".MainActivity">
+        <activity android:name=".activities.AuthActivity">
```

Comment:
Change the base package path to include activities so we can just specify the activity name